servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSResources.java

```diff
@@ -38,6 +41,9 @@ public Domain getDomain(@PathParam("domain") String domain) {
                 System.err.println("*** Warning: undeclared exception (" + code + ") for resource getDomain");
                 throw typedException(code, e, ResourceError.class);
             }
+        } finally {
+            this.delegate.stopTiming(timerMetric, "GET", code, "getDomain", context);
+            this.delegate.incrementMetric("GET", code, "getDomain", context);
```

Comment:
Also see if you can combine these two calls into one generic update metric call and then internally we can make two separate calls - one for increment and one for the timer. at least the code generation might look simpler